Library/.rubocop.yml

```diff
@@ -8,7 +8,15 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
As @reitermarkus pointed out, due to `Include` in the `.rubocop.yml` the cops won't be run on personal/custom Taps and the files directly passed as arguments which do not match the pattern under `Include`.
 To resolve this issue I need some clarification.  `brew style` command, runs on all the files inside `HOMEBREW_LIBRARY_PATH` while `brew audit` runs only on the forumlae files. 
Do we want(or is it okay for) the custom cops to get executed under `brew audit` and `brew style` also?